.github/workflows/build.yml

```diff
@@ -7,8 +7,32 @@ on:
       - litmus-v2
 
 jobs:
+  changes:
+    runs-on: ubuntu-latest
+    # Set job outputs to values from filter step
+    outputs:
```

Comment:
Here, Outputs might not be required.

Reply:
Actually, outputs are required if we trigger jobs not steps.
Ref: https://github.com/dorny/paths-filter#conditional-execution

Reply:
Okk yes @Jasstkn, Sorry my mistake, I thought they were jobs.